internal/envoy/shutdown/shutdown.go

```diff
@@ -24,20 +24,8 @@ import (
 	"github.com/tetratelabs/getenvoy/internal/envoy"
 )
 
-// EnableAll enables all debug options and returns a possibly empty list of errors.
-func EnableAll(r *envoy.Runtime) []error {
-	var errs []error
-	for _, debug := range []func(*envoy.Runtime) error{
-		enableEnvoyAdminDataCollection,
-		enableEnvoyLogCollection,
-		enableNodeCollection,
-	} {
-		if err := debug(r); err != nil {
-			errs = append(errs, err)
-		}
-	}
-	return errs
-}
+// EnableHooks is a list of functions that enable hooks in this package
```

Comment:
> in this package

curious, is there any other "hooks" outside this package?

Reply:
no there aren't.. I'll take out the unnecessary words!